PlasoScaffolder.ini

```diff
@@ -0,0 +1,11 @@
+[project]
+name: PlasoScaffolder
+name_description: a scaffolder project for plaso, easing parser and plugin creation. 
+maintainer: Log2Timeline maintainers <log2timeline-maintainers@googlegroups.com>
+homepage_url: https://github.com/log2timeline/PlasoScaffolder
+description_short: Plugin and parser scaffolder for plaso.
+description_long: The plaso (log2timeline) scaffolder is a tool written to make
+    writing a new plugin or parser in plaso easier by asking the developer few
+    questions and automatically create all the boilerplate code necessary to
+    start the process.
+python3_only: true
```

Comment:
As joachim mentioned, there's no "python3_only", so please remove this

Reply:
well, there is now... or at least once the other PR gets approved

Reply:
we can hold off on this until that PR get's pushed in l2tdevtools

Reply:
PR pushed